definitions/tools/concordance.cwl

```diff
@@ -1,48 +1,64 @@
+#!/usr/bin/env cwl-runner
+
 cwlVersion: v1.0
 class: CommandLineTool
-label: "Concordance tool"
-baseCommand: ["python", "/opt/concordance/newConcordance.py"]
+label: "Concordance checking between Tumor and Normal BAM"
+baseCommand: ["somalier"]
 requirements:
-    - class: DockerRequirement
-      dockerPull: "mgibio/concordance"
     - class: ResourceRequirement
+      coresMin: 1
       ramMin: 4000
+      tmpdirMin: 10000
+    - class: DockerRequirement
+      dockerPull: "brentp/somalier"
+arguments: ["-o", "concordance"]
 inputs:
-    bam_1:
+    vcf:
         type: File
         inputBinding:
+            prefix: "-s"
+            position: 1
+    threads:
```

Comment:
It could be problematic to allow this parameter to have a higher value than the number of cores specified in the `ResourceRequirement`.

Reply:
So then should i remove this input all together?